README.md

```diff
@@ -38,7 +37,7 @@ to assemble received octets, thus promoting completed octets to the octet
 buffer, freeing fields in the edge detection buffer.
 
 Look at the swsertest.ino example. There, on reset, ASCII characters ' ' to 'z'
-are send. This happens not as a block write, but in single write calls per
+are sent. This happens not as a block write, but in single write call per
```

Comment:
Thanks for all the writing fixes - but calls in plural is correct here, isn't it?

Reply:
Meant to write "a single write call" which I think is slightly more clear.